qap.go

```diff
@@ -32,7 +33,7 @@ func positiveReminder(a, b int) (result int) {
 	return
 }
 
-func createNeighbours(assignment Assignment, m1 IntMat, m2 IntMat, startIndex int) (result [neighbourCount]Assignment, costs [neighbourCount]int) {
+func createNeighbours(assignment Assignment, m1 IntMat, m2 IntMat, previousCostMatrix IntMat, previousCost int, startIndex int) (result [neighbourCount]Assignment, costs [neighbourCount]int) {
```

Comment:
Dużo paramków. ponieważ m1 i m2 są stałe na potrzeby instancji to zastanawiam się, czy nie byłoby spoko zrobić funkcję zwracającą funkcję
func(m1, m2 IntMat) (func (assignment Assignment, previousCostMatrix IntMat, previousCost int, startIndex int) (result [neighbourCount]Assignment, costs [neighbourCount]int))

Reply:
ale to nie ma znaczenia